taskcluster/app_services_taskgraph/target_tasks.py

```diff
@@ -10,6 +10,14 @@
 
 logger = logging.getLogger(__name__)
 
+
+def filter_out_shipping_phase(task):
+    return (
+        task.attributes.get("nightly")
+        or task.attributes.get("shipping_phase") in {None, "build"}
+    )
```

Comment:
Could you add a docstring here saying what "filter" means here?  I was initially confused since when this returns `True` the task is not filtered out.